README.md

```diff
@@ -361,6 +361,12 @@ When defining event filters, note the following:
 
 In the above example, all create, delete or update entity events to the `course_options` table and `id` matching value `1234` will be logged, or any import entity events to the `courses` table will also be logged.
 
+**IMPORTANT**:
+
+Please ensure you are not logging sensitive data to debug. Your project should define blocklist and pii fields, so these should prevent any sensitive data appearing in the events.
```

Comment:
@asatwal is it worth expanding pii to personally identifiable information? Also possibly link to the documentation for the blocklist and analytics_pii.yml here?